.ci_local_test/ros2_openvino_toolkit_test/test_cases/ros2_openvino_tool_model_download.sh

```diff
@@ -15,6 +15,13 @@ cd ~/catkin_ws/src/ros2_openvino_toolkit/data/model_list && sed -i 's/mask_rcnn_
 
 cd ~/catkin_ws/src/ros2_openvino_toolkit/data/model_list && omz_converter --list convert_model.lst -d /opt/openvino_toolkit/models/ -o /opt/openvino_toolkit/models/convert
 
+omz_downloader --name mask_rcnn_inception_resnet_v2_atrous_coco  -o /opt/openvino_toolkit/models/
```

Comment:
is it better to update these models into file download_model.lst and file convert_model.lst?